src/Traits/HasSettingsField.php

```diff
@@ -74,6 +74,14 @@ public function getSettingsFieldName(): string
         return $this->settingsFieldName ?? config('model_settings.settings_field_name');
     }
 
+    /**
+     * @return string
+     */
+    public function getConnectionName(): string
```

Comment:
This function already is part of Laravel, so why is this needed here?

Reply:
I see now! `Scheme::connection()` needs a value and `null` is not valid. So there must be a fallback value.